File: fix-fonts.js
const fs = require('fs');
const file = 'src/styles/global.css';
let content = fs.readFileSync(file, 'utf8');

const replacements = [
  ['0.72rem', '0.85rem'],
  ['0.75rem', '0.85rem'],
  ['0.78rem', '0.85rem'],
  ['0.8rem', '0.9rem'],
  ['0.82rem', '0.9rem'],
  ['0.85rem', '0.95rem'],
  ['0.88rem', '0.95rem'],
  ['0.9rem', '1rem'],
  ['0.92rem', '1rem'],
  ['0.95rem', '1.05rem'],
];

replacements.forEach(([from, to]) => {
  // Regex to match font-size: 0.xxrem;
  const regex = new RegExp(`font-size:\\s*${from};`, 'g');
  content = content.replace(regex, `font-size: ${to};`);
});

// Also replace in inline styles if any in TSX files
fs.writeFileSync(file, content);
console.log('Fixed CSS');

File: fix-fonts.py
import re

file_path = 'src/styles/global.css'
with open(file_path, 'r') as f:
    content = f.read()

replacements = [
  (r'font-size:\s*0\.72rem;', 'font-size: 0.85rem;'),
  (r'font-size:\s*0\.75rem;', 'font-size: 0.85rem;'),
  (r'font-size:\s*0\.78rem;', 'font-size: 0.85rem;'),
  (r'font-size:\s*0\.8rem;', 'font-size: 0.9rem;'),
  (r'font-size:\s*0\.82rem;', 'font-size: 0.9rem;'),
  (r'font-size:\s*0\.85rem;', 'font-size: 0.95rem;'),
  (r'font-size:\s*0\.88rem;', 'font-size: 0.95rem;'),
  (r'font-size:\s*0\.9rem;', 'font-size: 1rem;'),
  (r'font-size:\s*0\.92rem;', 'font-size: 1rem;'),
  (r'font-size:\s*0\.95rem;', 'font-size: 1.05rem;')
]

for old, new in replacements:
    content = re.sub(old, new, content)

with open(file_path, 'w') as f:
    f.write(content)

print("Fixed CSS fonts")
